projects/kit/components/tabs/tabs/tabs.component.ts

```diff
@@ -124,6 +128,20 @@ export class TuiTabsComponent implements AfterViewChecked {
 
         this.activeItemIndex = index;
         this.activeItemIndexChange.emit(index);
+    }
+
+    @HostListener('keydown.arrowRight.prevent', ['$event.target', '1'])
+    @HostListener('keydown.arrowLeft.prevent', ['$event.target', '-1'])
+    onKeyDownArrow(current: HTMLElement, step: number) {
+        const {tabs} = this;
+
+        moveFocus(tabs.indexOf(current), tabs, step);
+    }
+
+    updateScrollPosition(element: HTMLElement) {
+        if (!element) {
```

Comment:
This check doesn't make sense in terms of type, please add '?' to element argument.